### PR TITLE
Add regression test for #1422

### PR DESCRIPTION
**Pull request by theothornhill**
While I was hanging around this repo I though "why not pick some more stuff here".

So here is a quick and simple regression test for #1422.

The returned output reflects fantomas-tools as of e1054c1501b4eadb5fefa7b08daa8fb0f2b6153a

### CODE REVIEW

File: src/Fantomas.Tests/LetBindingTests.fs [hunk 1]
```diff
@@ -1557,3 +1557,43 @@ let myFun (a: decimal) b c : decimal = a + b + c
 let expensiveToCompute : int = 0
 let myFun (a: decimal) b c : decimal = a + b + c
 """
+
+[<Test>]
```
Comment (nojaf) [inline src/Fantomas.Tests/LetBindingTests.fs:1561]: Could you move this test to https://github.com/fsprojects/fantomas/blob/master/src/Fantomas.Tests/DotGetTests.fs please?

**Comment by nojaf** [comment]
>  "why not pick some more stuff here".

You are much too kind good sir!

File: src/Fantomas.Tests/LetBindingTests.fs (hunk 1 above)
Comment (theothornhill) [reply src/Fantomas.Tests/LetBindingTests.fs]: Absolutely. Done :) rebased

**Review by nojaf — APPROVED** [review]
Many thanks!

**Pull request merged**